src/engine/SCons/PathList.py

```diff
@@ -128,8 +128,13 @@ def subst_path(self, env, target, source):
         result = []
         for type, value in self.pathlist:
             if type == TYPE_STRING_SUBST:
-                value = env.subst(value, target=target, source=source,
-                                  conv=node_conv)
+                # We override conv below to use absolute paths when possible.
```

Comment:
Why make a global change like this. You can pass conv in in your call to subst_path right?

Reply:
I don’t have a call to subst_path. The test code I wrote uses vanilla PathList, which uses this code.

Reply:
So with this change if any code calls subst_path and sets conv, conv will be ignored.. that's not a change I can merge.

Reply:
I am not following. How would a caller pass conv to subst_path? The parameters are self, env, target, and source. node_conv is a function in this module, not a parameter.

Reply:
We discussed this and Slack and I confirmed that node_conv is not a parameter to this function, so it's not a global change.